### ISSUE TITLE
A JavaScript error occurred in the main process

### ISSUE THREAD

**Issue by kawaiier** [usebruno/bruno#749]
Hello!

I've just downloaded Bruno (`bruno 0.27.1 x64 mac.dmg`) for the first time on my x64 Macbook with Ventura 13.6 (22G120). When trying to open the app I get this message:

```
Uncaught Exception:
Error: Cannot find module 'chai-string'
Require stack:
- /Applications/Bruno.app/Contents/Resources/app.asar/node_modules/@usebruno/js/src/runtime/assert-runtime.js
- /Applications/Bruno.app/Contents/Resources/app.asar/node_modules/@usebruno/js/src/index.js
- /Applications/Bruno.app/Contents/Resources/app.asar/src/ipc/network/index.js
- /Applications/Bruno.app/Contents/Resources/app.asar/src/index.js
- 
at Module._resolveFilename (node:internal/modules/cjs/loader:940:15)
at n._resolveFilename (node:electron/js2c/browser_init:249:1105)
at Module._load (node:internal/modules/cjs/loader:785:27)
at c._load (node:electron/js2c/asar_bundle:5:13339)
at Module.require (node:internal/modules/cjs/loader:1012:19)
at require (node:internal/modules/cjs/helpers:102:18)
at Object.<anonymous> (/Applications/Bruno.app/Contents/Resources/app.asar/node_modules/@usebruno/js/src/runtime/assert-runtime.js:10:10)
at Module._compile (node:internal/modules/cjs/loader:1120:14)
at Module._extensions..js (node:internal/modules/cjs/loader:1175:10)
at Module.load (node:internal/modules/cjs/loader:988:32)
```

I can still press **OK** button and proceed, but there are no windows on the screen even though the app is running in the Dock

**Comment by bzh4bzh** [comment]
Same error on Win64

**Comment by jwetzell** [comment]
I just opened a PR that should fix this issue but will require the Bruno to cut a new release (hopefully take the 0.27.1 release down as well).

**Comment by stacy-rendall** [comment]
Error on Win64:
```
Error: Cannot find module 'chai-string'
```
This previous commit seems related: https://github.com/usebruno/bruno/commit/5ba3903c31790d057eb8a9e75abf7d74be20d8fc

**Comment by Its-treason** [comment]
test
test
test
test
test
test
test
test
test
test
test

**Comment by helloanoop** [comment]
@jwetzell @stacy-rendall @kawaiier Apologies for the hiccup.

@jwetzell

**Comment by jwetzell** [comment]
@helloanoop just downloaded and it works great!

**Comment by helloanoop** [comment]
This issue is only present on `v0.27.1`

#### Windows Users:
Please upgrade to the latest binary `v0.27.2`

#### Linux Users:
Snap, Deb builds + Linux binaries are available for `v0.27.2`

#### Mac Users
Binaries are available for `v0.27.2`
If you are using homebrew, please use `brew install bruno@0.27.0` until the hombrew cask gets updated with the fix

**Comment by jwetzell** [comment]
Yes this has been resolved in latest release.
